MonstieWash/Assets/Scripts/Traversal/TestTravObj.cs

```diff
@@ -0,0 +1,46 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class TestTravObj : MonoBehaviour, INavigator
+{
```

Comment:
### 💡 **Suggestion**  | _This is a suggestion, feel free to take it or leave it._

Also might be worth renaming this class to something more appropriate since it's no longer just a test.

Reply:
# oops i forgetti